source/Guides/ros2_tracing_trace_and_analyze.md

```diff
@@ -47,6 +47,12 @@ $ git clone https://gitlab.com/ros-tracing/tracetools_analysis.git
 $ cd ..
 ```
 
+Install dependencies with rosdep.
+```
+$ rosdep update
+$ rosdep install --from-paths src --ignore-src -y --skip-keys "fastcdr rti-connext-dds-6.0.1 urdfdom_headers"
```

Comment:
@ijnek, do we need to skip these keys?

Reply:
I'm actually not quite sure what the reason is, I've posted [a question](https://answers.ros.org/question/402748/why-does-ubuntu-source-installations-rosdep-skip-some-keys/) on ROS answers, so let's see if someone can fill us in.

Reply:
So - if it's not a dependency for ROS2 Rolling according to the tutorials, I guess we could keep it the same way as the tutorials.

Reply:
Quoting the answer to the question:

> Connext is skipped because it needs a license to run so most people won't want to take the time to build it if they can't run it. Instructions for installing it directly are below.
> 
> ``urdfdom_headers`` and ``fastcdr`` are skipped because they do not provide a package.xml in their installed targets so rosdep via catkin_pkg doesn't detect that they've been installed, even if they've been installed or are in the workspace.

I think it's better to leave it as is and skip those keys.

Reply:
Agreed!